source/educators/index.rst

```diff
@@ -1,76 +1,182 @@
 Open edX Educators
 ##################
 
-.. grid:: 1 2 2 2
+.. grid:: 1 2 2 2 
    :gutter: 3
    :padding: 0
 
    .. grid-item-card:: Quick Starts
       :class-card: sd-shadow-md sd-p-2
       :class-footer: sd-border-0
+      :columns: 12
 
-      * :doc:`quickstarts/build_a_course`
-      +++
-      .. button-ref:: quickstarts/index
+      | :ref:`Quick Start Build a Course`
+      | :ref:`What is LMS`
+      | :ref:`What is Studio`
+
+   .. grid-item-card:: Creating a Course in Open edX®
+      :class-card: sd-shadow-md sd-p-2
+      :class-footer: sd-border-0
+
+      .. image:: /_images/educators_homepage/build_course_undraw.png
```

Comment:
Right now when you click on the images, it opens a link to just the image.  Use the 'target' parameter to link the image to the same target as the `See All` link for all the cards.

Reply:
Fixed in this commit: [d63d26c](https://github.com/openedx/docs.openedx.org/pull/601/commits/d63d26c611119162a69cdf4130c2619ca9c237d9)